tests/io/fixtures/pa_parser.xml

```diff
@@ -1,120 +1,119 @@
-<p>More than a third (35%) of school leaders said they typically experienced abuse from parents every month, and 16% said it happened weekly.</p>
-<p>One senior leader told how "malicious and vexatious complaints made me want to leave my job and made me ill".</p>
-<p>Another said: "I felt very low and dreaded opening my emails ... another member of the team said they might have to resign to avoid reading the emails."</p>
-<p>The NAHT is calling for the Government to send a clear signal to parents that abuse is unacceptable in schools.</p>
-<p>The union wants ministers to conduct an urgent review of complaints procedures to deter a "vexatious" use of the existing system by parents.</p>
-<p>When asked why school leaders are facing a rise in abuse from parents, Mr Whiteman said: "It's hard to pinpoint a single reason for the increase.</p>
-<p>"Schools are certainly seeing more pupils and parents struggling after a decade of under-investment under previous governments in social care, mental health services and support for children with special educational needs.</p>
-<p>"This happened alongside cuts to welfare support, and we then had the impact of the pandemic and inflation.</p>
-<p>"Parents need support but there is no excuse to take out their frustration on school staff doing their level best to help, often without the resources or access to the specialist services they need."</p>
-<p>The NAHT represents school leaders working across the education sector, including: headteachers, executive heads, trust chief executives, deputy and assistant heads, vice principals and school business leaders.</p>
-<p>A <org>Department for Education</org> (DfE) spokesperson said: "Nobody should face violence or abuse in the workplace, including our incredible school staff who are vital to improving life chances for all children.</p>
-<p>"As part of our Plan for Change, we are committed to turning the tide on poor behaviour, breaking down barriers to opportunity and ensuring every child can achieve and thrive.</p>
-<p>"Our new regional improvement teams will work with schools to spread the highest standards of behaviour across our classrooms, and we are committed to providing access to a specialist mental health professional in every school.</p>
-<p>"But we know there is more to do and are looking closely at how we can go further to support teachers and drive high and rising school standards for all our children."</p>
+<p>The programme for the Edinburgh Festival Fringe has been launched, including more than 3,350 shows across 265 venues.</p>
+<p>
+Comedy, music, dance, children's shows, magic and cabaret will all be part of the arts festival, which runs from
+<chron>August 1 to 25</chron>
```

Comment:
can you add a test checking that we don't have those custom tags in html? things like `chron org person location`